test/addons/async-hello-world/test.js

```diff
@@ -4,7 +4,7 @@ var assert = require('assert');
 const binding = require(`./build/${common.buildType}/binding`);
 
 binding(5, common.mustCall(function(err, val) {
-  assert.equal(null, err);
-  assert.equal(10, val);
+  assert.strictEqual(null, err);
```

Comment:
@lpinca Swapped this instance and a couple others that were in expected, actual order instead of the desired actual, expected signature. Then rebased and squashed changes into one commit.